### DISCUSSION TITLE
add donate button in crypto if you do not mind

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [beerisgood/Windows11_Hardening#7] [General]
hello, that is resource i was trying to find

**Comment by beerisgood** [comment]
I am not a friend of crypto.
If interested, i can once look at normal donation provider.

**Reply by ghost** [reply]
I don't mind donating either, but only using crypto, I don't love crypto nor do I want to make profit off of it, but it's a payment system where you stay anonymous, you aren't hindered by 3rd party rules, complications, and no country can prevent you from paying with this payment system because no single country owns it, which is exactly the opposite of visa/master cards, paypall etc. 
crypto payment doesn't discriminate against anyone, no matter their nationality, background or identify, that's why I use it.

**Reply by beerisgood** [reply]
This does not make you anonymous either. 
Since I do not want to support crypto in its current (mining) form and a non-crypto method is not desired, I close here for now.

**Comment by beerisgood** [comment]
@dzmitry-lahoda I added different support platforms to my repositories 🍻
